### DISCUSSION TITLE
Importing problem with gob 4.0.2 and zbrush 2024.0.1.

### DISCUSSION THREAD

**Discussion by Pie-Ric** [JoseConseco/GoB#456] [Q&A]
Hi guys,
I use version of gob 4.0.2 and zbrush 2024.0.1.
When I export from zbrush to blender (after activating import to blender) it makes me export only the first subtool automatically and the ones after that don't. To import them into blender I have to use the manual button and I still get this error. Where am i going wrong? thank you very much 
![Screenshot 2023-12-22 115238](https://github.com/JoseConseco/GoB/assets/154597463/8f5d0c20-6992-48ad-b147-20c89e6c9e66)

**Comment by gastonhoz** [comment]
the same thing happens to me, when I want to export from zbrush to blender, it only exports a single subtool. I couldn't solve it!
![IMG_20240211_022235_631](https://github.com/JoseConseco/GoB/assets/159724144/98ad7303-1c47-4bff-890e-c43dee19f440)

**Comment by dsfsdfas234** [comment]
same issue
![image](https://github.com/JoseConseco/GoB/assets/101152094/e78cc813-6e43-4aa4-adb2-650101a83bb1)
